ci/coverity-scan.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+# Copyright 2018 Google LLC
```

Comment:
Do you typically add a blank line to separate `#!` line from the license or no?

Reply:
More often than not we put an empty comment line.